hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieCDCFileIndex.scala

```diff
@@ -79,8 +79,4 @@ class HoodieCDCFileIndex (override val spark: SparkSession,
       new Path(fileGroupId.getPartitionPath, fileGroupId.getFileId).toString
     }.toArray
   }
-
-  override def getRequiredFilters: Seq[Filter] = {
-    Seq.empty
-  }
```

Comment:
Should this be kept for CDC file index?

Reply:
yeah. Added to the description, but this is ontop of https://github.com/apache/hudi/pull/13527